### DISCUSSION TITLE
Lidar scan generates some ghost points

### DISCUSSION THREAD

**Discussion by ajay1606** [IPG-Automotive-UK/Aslan-CarMaker#10] [General]
Hello,

While working with LiDAR sensor data in the Simulation environment observed some ghost points in the lidar scan though actually there was no object present.

Case 1:
![image](https://user-images.githubusercontent.com/29722843/185526782-f07fdf0b-99f7-4211-9bce-a29d09439adf.png)

Case 2:
![image](https://user-images.githubusercontent.com/29722843/185526909-1e9e833f-915f-4c12-9427-e8abe8ccb17d.png)

Any inputs to fix this issue? Much appreciated. 

Regards,
Ajay

**Comment by bogotm** [comment]
Hello, is this IPG Movie? This level of ghosting should not exist in CarMaker alone, if it's rViz then it might have something to do with the frame or reference in which the LiDAR data is being displayed in ROS.
